### PR TITLE
Split merge

### PR DESCRIPTION
**Pull request by Jonybang**

### CODE REVIEW

File: contracts/SplitMerge.sol [hunk 1]
```diff
@@ -4,32 +4,115 @@ pragma experimental "v0.5.0";
...
+  function setPlotManager(PlotManager _plotManager) public onlyOwner {
+    plotManager = _plotManager;
+  }
+
+  modifier ownerOrPlotManager() {
+    require(plotManager == msg.sender || owner == msg.sender, "No permissions to mint geohash");
+    _;
+  }
+
+  function mintGeohash(uint256 _geohashToken) public ownerOrPlotManager {
```
[the remark this answers is not in the document]
Comment (Jonybang) [reply contracts/SplitMerge.sol]: Maybe it's check should be inside SpaceToken.mint?

[the remark this answers is not in the document]
Comment (chebykin) [reply contracts/SplitMerge.sol]: SpaceToken.mint could mint both geohash and pack tokens, geohash/pack identifier should be already defined at that moment.

[the remark this answers is not in the document]
Comment (Jonybang) [reply contracts/SplitMerge.sol]: added todo

**Comment by Jonybang** [comment]
@chebykin all code fixed and all tests passed

**Pull request merged**